proprietary/tokens/src/components/amsterdam/switch.tokens.json

```diff
@@ -0,0 +1,46 @@
+{
+  "amsterdam": {
+    "switch": {
+      "font-family": {
+        "value": "{amsterdam.typography.font-family}"
+      },
+      "background-color": {
+        "value": "{amsterdam.color.neutral-grey3}"
+      },
+      "toggle": {
+        "background-color": {
+          "value": "{amsterdam.color.primary-white}"
+        },
+        "width": {
+          "value": "58px"
```

Comment:
This should be in rems I think, and probably related to the circle size. If we use a text level line-height for the circle size, this should be twice that, I think?